Source/controls/touch/renderers.cpp

```diff
@@ -516,30 +491,100 @@ void VirtualGamepadRenderer::UnloadArt()
 {
 	menuPanelRenderer.UnloadArt();
 	directionPadRenderer.UnloadArt();
-	buttonArt.clear();
-	potionArt.clear();
+	buttonArt.clearSurface();
+	potionArt.clearSurface();
 }
 
 void VirtualMenuPanelRenderer::UnloadArt()
 {
-	menuArt.clear();
-	menuArtLevelUp.clear();
+	menuArt.clearSurface();
+	menuArtLevelUp.clearSurface();
 }
 
 void VirtualDirectionPadRenderer::UnloadArt()
 {
-	padArt.clear();
-	knobArt.clear();
+	padArt.clearSurface();
+	knobArt.clearSurface();
 }
 
-void InitVirtualGamepadGFX(SDL_Renderer *renderer)
+void InitVirtualGamepadGFX()
 {
-	Renderer.LoadArt(renderer);
+	Renderer.LoadArt();
 }
 
 void FreeVirtualGamepadGFX()
 {
 	Renderer.UnloadArt();
 }
 
+#ifndef USE_SDL1
+void VirtualGamepadRenderer::createTextures(SDL_Renderer &renderer)

```

Comment:
These files don't get compiled in with SDL1.
https://github.com/diasurgical/devilutionX/blob/d8dde8d2b119dfbc7973384aa598bd9e9de2c864/Source/CMakeLists.txt#L254-L260

The only reason there's a distinction between texture rendering vs surface rendering is because of the `Upscale` option.

Reply:
Ah, thanks! Done